app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -1963,11 +2004,14 @@ object TelemetryWrapper {
 
         init {
             lazyInit()
+            Log.d(TAG, "EVENT:$category/$method/$`object`/$value")
```

Comment:
Not sure if intended to keep these log.

Reply:
yes. So developers can debug without setting the break point.